Project 1/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>  
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel= "stylesheet" href= "styles.css">
+    <title>Intelligent Octopus</title>
+</head>
+<body>
+        <body style="background-color:purple;">
```

Comment:
All your styles should be included in your `styles.css`
```suggestion
        <body>
```